src/__main__.py

```diff
@@ -24,59 +29,99 @@ def inner(*args, **kwargs):
                     return error_message
             else:
                 raise e
+
     return inner
 
+
+def show_help():
+    # Print available commands with color highlighting
```

Comment:
use doc block instead of comment:
 '''
 Print available commands with color highlighting
  '''